src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/PrisonResource.kt

```diff
@@ -70,4 +70,10 @@ class PrisonResource(
     @Schema(description = "Offender Number", example = "A12345", required = true)
     @PathVariable offenderNumber: String
   ) = prisonService.getPrisonerImage(offenderNumber)
+
+  @GetMapping(value = ["/temporary-absences/{agencyId}"], produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
will need to add the response annotations as above for the swagger doc

Reply:
done